### PR TITLE
[Deprecation]LPS-103868

### PR DESCRIPTION
**Pull request by tinatian**

### CODE REVIEW

**Comment by tinatian** [comment]
ci:test

**Comment by shuyangzhou** [comment]
ci:test:gauntlet

**Comment by shuyangzhou** [comment]
@vicnate5 another full scan pull, thanks!

**Comment by vicnate5** [comment]
@shuyangzhou No regressions found in automation.

**Comment by shuyangzhou** [comment]
The boxing part is not needed, please resend.

File: modules/apps/asset/asset-categories-admin-web/src/main/java/com/liferay/asset/categories/admin/web/internal/exportimport/data/handler/AssetCategoryStagedModelDataHandler.java [hunk 1]
```diff
@@ -281,8 +281,8 @@ protected void doImportStagedModel(
 		categoryUuids.put(category.getUuid(), importedCategory.getUuid());
 
 		portletDataContext.importPermissions(
-			AssetCategory.class, category.getCategoryId(),
-			importedCategory.getCategoryId());
+			AssetCategory.class, Long.valueOf(category.getCategoryId()),
```
Comment (tinatian) [inline modules/apps/asset/asset-categories-admin-web/src/main/java/com/liferay/asset/categories/admin/web/internal/exportimport/data/handler/AssetCategoryStagedModelDataHandler.java]: Hi @dantewang @yuhai ,
You actually do not need to do this convert, it should autobox. As long as you removed the deprecate method, this is going to link to the non-deprecate one.
Please fix all the changes like this and resend.

Thanks.
Tina.

**Pull request closed without merging**